MIP101/MIP101.md

```diff
@@ -187,11 +185,11 @@ AllocatorDAOs are the most prominent SubDAOs. They can generate Dai Directly fro
 
 #### MiniDAO
 
-MiniDAOs are created by AllocatorDAOs through a simple Governance decision. MiniDAOs enable AllocatorDAOs to innovate many new products and ideas they can then diversify and decentralize by spinning them off as MiniDAOs. MiniDAOs can be very diverse and in some cases very short lived as they try out a new idea that may or may not hit product market fit. Because of their extremely low barrier to entry and ease of creation they can be highly experimental and creative, allowing some of them to hit surprising home runs.
+MiniDAOs are created by AllocatorDAOs through a simple Governance decision. MiniDAOs enable AllocatorDAOs to innovate many new products and ideas they can then diversify and decentralize by spinning them off as MiniDAOs. MiniDAOs can be very diverse and in some cases very short lived as they try out a new idea that may or may not hit product market fit. Because of their extremely low barrier to entry and ease of creation they can be ****highly experimental and creative, allowing some of them to hit surprising home runs.
```

Comment:
```suggestion
MiniDAOs are created by AllocatorDAOs through a simple Governance decision. MiniDAOs enable AllocatorDAOs to innovate many new products and ideas they can then diversify and decentralize by spinning them off as MiniDAOs. MiniDAOs can be very diverse and in some cases very short lived as they try out a new idea that may or may not hit product market fit. Because of their extremely low barrier to entry and ease of creation they can be highly experimental and creative, allowing some of them to hit surprising home runs.
```